### DISCUSSION TITLE
docker snap stable channel is not docker stable release

### DISCUSSION THREAD

**Discussion by kinghat** [canonical/docker-snap#75] [General]
it seems that the snap stable channel is not kept up to date with the stable releases of docker engine. in the [docs](https://github.com/docker-snap/docker-snap#installation) it says:

> To install the latest stable release of Docker CE using snap:

docker `20.10.17` is out and docker snap stable channel has been on `20.10.14` for a while. should the docs be changed to use `--edge` to be on docker stable releases?

**Comment by tianon** [comment]
@mclemenceau :sweat_smile:

20.10.17 is in edge - it's probably prudent that we look to promote that through :+1:

(We probably also want to update the docs to be a bit more accurate to the reality that the stable channel won't _always_ be exactly the latest release of Docker.)

**Reply by kinghat** [reply]
thanks for the reply @tianon, to get a better idea of the docker snap release flow, what is the process or requirements for moving docker stable release to the docker snap stable release channel?

**Reply by tianon** [reply]
Canonical has a suite of tests they run internally to validate new builds before promoting them through to the stable channel.

**Reply by kinghat** [reply]
is that automated? docker `20.10.14` was released on [2022-03-23](https://docs.docker.com/engine/release-notes/#201014) and `20.10.17` was released [2022-06-06](https://docs.docker.com/engine/release-notes/#201017).

**Comment by xnox** [comment]
a newer update is in progress getting validated.

whilst validation is automated, fixing the fallout from updates is not hence it takes time.
